showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITCrud.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.showcase.v1beta1.it;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import com.google.showcase.v1beta1.CreateUserRequest;
+import com.google.showcase.v1beta1.DeleteUserRequest;
+import com.google.showcase.v1beta1.EchoSettings;
+import com.google.showcase.v1beta1.IdentityClient;
+import com.google.showcase.v1beta1.IdentitySettings;
+import com.google.showcase.v1beta1.ListUsersRequest;
+import com.google.showcase.v1beta1.ListUsersResponse;
+import com.google.showcase.v1beta1.UpdateUserRequest;
+import com.google.showcase.v1beta1.User;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ITCrud {
+
+  private static final User DEFAULT_USER =
+      User.newBuilder()
+          .setDisplayName("Jane Doe")
+          .setEmail("janedoe@example.com")
+          .setNickname("Doe")
+          .setHeightFeet(5)
+          .setAge(25)
+          .build();
+
+  private IdentityClient httpJsonClient;
+
+  @Before
+  public void setup() throws GeneralSecurityException, IOException {
+    // Create HttpJson IdentityClient
+    IdentitySettings httpJsonIdentitySettings =
```

Comment:
If there is no special settings, can we move this to [TestClientInitializer](https://github.com/googleapis/gapic-generator-java/blob/main/showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/util/TestClientInitializer.java) for better reusability?

Reply:
Sure, done.